src/Yasumi/Provider/Ukraine.php

```diff
@@ -83,24 +168,6 @@ private function calculateChristmasDay(): void
         ));
     }
 
-    /**
-     * International Workers' Day.
```

Comment:
If a holiday is not celebrated observed any longer, please don't delete it but provide a condition that it has been ended since a particular date/year.

Reply:
Reasonable!